### PR TITLE
Update DataConsumerWithSequencerCheck.sol

### PR DESCRIPTION
**Pull request by aviggiano**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.Any change needs to be discussed before proceeding.**

## Closing issues

Closes https://github.com/smartcontractkit/documentation/issues/1994

## Description

Updates code sample to match documentation

## Changes

- After a recent security review, an auditor pointed out that the L2 Sequencer Feed code sample does not match the recommendation from the docs
- This PR fixes the code sample to match the documentation
- See https://github.com/code-423n4/2024-06-size-findings/issues/209

### CODE REVIEW

**Comment by dwightjl** [comment]
Thank you for opening this PR. I'm confirming how best to handle situations where `startedAt==0`. I'm not sure if an invalid round should cause `isSequencerUp` to be `false`.

**Comment by dwightjl** [comment]
My understanding is that correctly handling `startedAt == 0` is application-dependent, but it is good to highlight that it is available. I'll get a review on this change.

File: src/content/data-feeds/l2-sequencer-feeds.mdx [hunk 1]
```diff
@@ -101,6 +101,6 @@ The `sequencerUptimeFeed` object returns the following values:
 - `answer`: A variable with a value of either `1` or `0`
```
Comment (thedriftofwords) [inline src/content/data-feeds/l2-sequencer-feeds.mdx:101]:
```suggestion
- `answer`: A variable with a value of either `0` or `1`
```

File: public/samples/DataFeeds/DataConsumerWithSequencerCheck.sol [hunk 2]
```diff
@@ -15,6 +15,7 @@ contract DataConsumerWithSequencerCheck {
 
     uint256 private constant GRACE_PERIOD_TIME = 3600;
 
+    // error InvalidUptimeFeedRound();
```
Comment (thedriftofwords) [inline public/samples/DataFeeds/DataConsumerWithSequencerCheck.sol:18]: Are we keeping this commented out line for any particular reason?

**Review by thedriftofwords — CHANGES_REQUESTED** [review]

**Comment by dwightjl** [comment]
Still getting confirmation on some changes here.

**Review by dwightjl — APPROVED** [review]

**Pull request merged**